cmd/mito/main.go

```diff
@@ -15,6 +15,31 @@
 // specific language governing permissions and limitations
 // under the License.
 
+// The mito executable is a CEL program evaluation tool that allows development
+// of Filebeat CEL input integrations without the need to run have a running
+// stack. Not all feature of the Filebeat input are present, but most behaviors
+// that are needed to mimic an integration are available.
+//
+//	Usage of mito:
+//
+//	  mito [opts] <src.cel>
+//
+//	  -cfg string
+//	    	path to a YAML file holding configuration for global vars and regular expressions
```

Comment:
Is there a struct that this can link to help users discover the possible config options?

Reply:
It's based around the config in the input, but what I'll do is export the [structs in mito](https://github.com/elastic/mito/blob/01487155530de7b27a1f6cb07633c1339cbd53ce/mito.go#L415-L447) and link to them.